### PR TITLE
Add expect_table_column_count_to_equal_other_table_times_factor

### PR DESCRIPTION
**Pull request by nysthee**
Hi

We have a use case where we have a table a and we want to validate that table b always has a row count that is equal to the row count in table a times a predefined factor.

### CODE REVIEW

File: integration_tests/models/schema_tests/schema.yml [hunk 1]
```diff
@@ -117,6 +117,9 @@ models:
             max_value: 4
         - dbt_expectations.expect_table_row_count_to_equal_other_table:
             compare_model: ref("data_test")
+        - dbt_expectations.expect_table_row_count_to_equal_other_table_times_factor:
+            compare_model: ref("data_test")
+            factor: 1
```
Comment (clausherther) [inline integration_tests/models/schema_tests/schema.yml:122]: Should we maybe introduce a new test column that is factored up something other than `1`?

**Review by clausherther — APPROVED** [review]
Hi @nysthee, thanks for submitting this, looks great. I added a small comment re: the integration, but I'll leave it up to you if you want to change that. Let me know either way and I'll go ahead and merge once you're ready.

**Comment by nysthee** [comment]
I agree, makes sense to make the test a bit more meaningful. I will do it and ping you once it’s there!

- - -
Thomas Nys
+32 475 269 452
https://calendly.com/thomasnys

On 9 April 2021 at 16:10:08, Claus Herther ***@***.***) wrote:

@clausherther approved this pull request.

Hi @nysthee, thanks for submitting this, looks great. I added a small comment re: the integration, but I'll leave it up to you if you want to change that. Let me know either way and I'll go ahead and merge once you're ready.

In integration_tests/models/schema_tests/schema.yml:

> @@ -117,6 +117,9 @@ models:
             max_value: 4
         - dbt_expectations.expect_table_row_count_to_equal_other_table:
             compare_model: ref("data_test")
+        - dbt_expectations.expect_table_row_count_to_equal_other_table_times_factor:
+            compare_model: ref("data_test")
+            factor: 1
Should we maybe introduce a new test column that is factored up something other than 1?

**Comment by nysthee** [comment]
hi @clausherther 

I added a new model with factor 2 number of rows of the original table (didn't bother with the content since we only care about the number of rows)

**Comment by clausherther** [comment]
@nysthee looks good, thanks!

**Comment by clausherther** [comment]
I will cut a new release this weekend, might wrap some other work into that. Thanks again for your contribution!

**Pull request merged**